src/popup/pages/Retip.vue

```diff
@@ -119,7 +120,7 @@ export default defineComponent({
     const formModel = ref<IFormModel>({
       amount: '',
     });
-
+    const api = useAeApiActions();
```

Comment:
I suggest to treat this a a regular composable, so pick the methods from returned object instead of creating the api constant.